dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -33,9 +41,10 @@ export const NoExpiringRuns = () => {
       <TextContent className="no-runs-wrapper">
         <Text component={TextVariants.h3}> You have no runs expiring soon</Text>
         <Text component={TextVariants.p}>
-          Runs that have expiration date within next 10days will appear here.
-          These runs will be automatically removed from the system if left
-          unacknowledged. <Button variant="link">Learn More.</Button>
+          Runs that have expiration date within next {EXPIRATION_DAYS_LIMIT}{" "}
```

Comment:
Is the forced space insert necessary here? That suggests JSX is somehow eating a real blank between the substitution text and "days"?? Huh.

Reply:
Yes, it seems odd that lines 44 and 46 need an explicit blank character when line 45 does not.  (Why does line 46 need it??)

Reply:
Actually, Line 46 does not require a blank character. But between the substitution text and "days" we need a blank character